tests/translator/input/error_function_invalid_api_event.yaml

```diff
@@ -0,0 +1,52 @@
+Resources:
+  FunctionApiNoMethod:
+    Type: 'AWS::Serverless::Function'
+    Properties:
+      CodeUri: s3://sam-demo-bucket/hello.zip
+      Handler: hello.handler
+      Runtime: python2.7
+      Events:
+        ApiEvent:
+          Type: Api
+          Properties:
+            method: get # NOTE: lowercase 'm'
+            Path: /
+  
+  FunctionApiNoPath:
+    Type: 'AWS::Serverless::Function'
+    Properties:
+      CodeUri: s3://sam-demo-bucket/hello.zip
+      Handler: hello.handler
+      Runtime: python2.7
+      Events:
+        ApiEvent:
+          Type: Api
+          Properties:
+            Method: get
+            path: / # NOTE: lowercase 'm'
+  
+  FunctionApiPathArray:
+    Type: 'AWS::Serverless::Function'
+    Properties:
+      CodeUri: s3://sam-demo-bucket/hello.zip
+      Handler: hello.handler
+      Runtime: python2.7
+      Events:
+        ApiEvent:
+          Type: Api
+          Properties:
+            Method: get
+            Path: [/, /users]
+  
+  FunctionApiMethodArray:
+    Type: 'AWS::Serverless::Function'
+    Properties:
+      CodeUri: s3://sam-demo-bucket/hello.zip
+      Handler: hello.handler
+      Runtime: python2.7
+      Events:
+        ApiEvent:
+          Type: Api
+          Properties:
+            Method: [get, post]
```

Comment:
People also attempt to use intrinsics, but that's also not a string so this should work fine.